modules/nw-create-load-balancer-service.adoc

```diff
@@ -38,15 +38,22 @@ spec:
   - name: db
     port: 3306 <2>
   loadBalancerIP:
-  type: LoadBalancer <3>
+  loadBalancerSourceRanges: 10.0.0.0/8,192.168.0.0/16 <3>
+  type: LoadBalancer <4>
   selector:
-    name: mysql <4>
+    name: mysql <5>
 ----
 <1> Enter a descriptive name for the load balancer service.
 <2> Enter the same port that the service you want to expose is listening on.
-<3> Enter `loadbalancer` as the type.
-<4> Enter the name of the service.
-
+<3> Enter a list of specific IP addresses to restrict traffic through the load balancer. This field will be ignored if the cloud-provider does not support the feature.
+<4> Enter `loadbalancer` as the type.
+<5> Enter the name of the service.
++
+[NOTE]
+====
+To restrict traffic through the load balancer to specific IP addresses, it is recommended to use the `service.beta.kubernetes.io/load-balancer-source-ranges` annotation rather than setting the `loadBalancerSourceRange` field.
```

Comment:
It's `loadBalancerSourceRange` here and `loadBalancerSourceRanges` in the YAML. Please double-check.

Reply:
@kalexand-rh  It should be loadBalancerSourceRanges - I have changed it.